client/src/components/main/sidebar/CurrentUser.js

```diff
@@ -17,20 +17,22 @@ function CurrentUser(props) {
 
   useEffect(() => {
     setIsLoading(true);
-    axios
-      .get(`/api/users/${localStorage.userId}`, {
-        headers: { authorization: `bearer ${localStorage.authToken}` },
-      })
-      .then(res => {
-        setIsLoading(false);
-        setLoggedinUser(res.data.name);
-        setImageUrl(res.data.userImage);
-      })
-      .catch(err => {
-        setIsLoading(false);
-        setIsError(true);
-        errorMessage.set_message([{ msg: 'Unable to get the user.' }]);
-      });
+    if (localStorage.loggedIn) {
```

Comment:
I tried to understand here. when a user don't logout, his/her localStorage.loggedIn is always true, how can will help to clear the storage when a user isn't authenticated.

Reply:
@Junjpan this is if the user has been deleted and not when a user logs out